src/main/kotlin/com/cognifide/gradle/aem/common/http/HttpClient.kt

```diff
@@ -177,6 +177,10 @@ open class HttpClient(val aem: AemExtension) {
         return Formats.asJson(asStream(response))
     }
 
+    fun asJson(jsonString: String): DocumentContext {
```

Comment:
Fold it to:
``` fun asJson(jsonString: String) = Formats.asJson(jsonString)```

Reply:
123

Reply:
I'd rather to leave it like this to be consistent with lines above and below

Reply:
if max line length is not exceeded, I prefer @mierzwid 's approach

Reply:
done